deploy/openshift/clusterrole.yaml

```diff
@@ -77,6 +77,7 @@ rules:
 - apiGroups: ["authentication.istio.io"]
   resources:
   - policies
+  - meshpolicies
```

Comment:
@jmazzitelli @gbaufake more changes in clusterrole.yaml
@xeviknal It needs also a JIRA in Maistra project too.

Reply:
Done!